05_debug/Makefile

```diff
@@ -0,0 +1,12 @@
+ifneq ($(KERNELRELEASE),)
+# kbuild part of makefile
+obj-m   := task05_sys.o
+else
+# normal makefile
+
+obj-m   := task05_sys.o
```

Comment:
You have two modules in this task.
Need to do following: either add line for second module here or add a separate makefile for each module.